### PR TITLE
Add missing class to Markdown

### PR DESCRIPTION
**Pull request by youngbob**
markdown.blockprocessors.ReferenceProcessor was missing from this stub.

https://github.com/Python-Markdown/markdown/blob/master/markdown/blockprocessors.py#L559

### CODE REVIEW

File: stubs/Markdown/markdown/blockprocessors.pyi [hunk 1]
```diff
@@ -62,4 +62,8 @@ class HRProcessor(BlockProcessor):
     match: Match[str]
 
 class EmptyBlockProcessor(BlockProcessor): ...
+
+class ReferenceProcessor(BlockProcessor):
+    RE: Pattern[str]
```
Comment (Akuli) [inline stubs/Markdown/markdown/blockprocessors.pyi:67]: Maybe this and most other things in this file should use `ClassVar`, but that can be changed in a separate PR if needed.

**Review by Akuli — APPROVED** [review]

**Review by srittau — APPROVED** [review]
Thanks!
